mlflow_project/README.md

```diff
@@ -0,0 +1,36 @@
+### Train and Publish Locally With MLFlow
+#### Jupyter Notebook Workflow
+[Jupyter Notebook](notebooks/rapids_mlflow_databricks_train_deploy.ipynb)
+
+#### CLI Based Workflow
+1. Create a new conda environment.
+    1. `$ conda create -f envs/conda.yaml`
+1. Train the model
+    1. `$ cd mlflow\_project`
+    1. The example project is described in [MLProject](https://www.mlflow.org/docs/latest/projects.html) file.
+        1. This can be edited to allow additional command line variables, specify conda environments, and training
+        parameters (see link for additional information).
+    1. Publish to local tracking server
+        1. Here we instruct mlflow to run our training routine locally, and publish the result's to the local file system.
+        1. ```shell script
+          $ mlflow run file:///$PWD -b local\
+                                   -P conda-env=$PWD/envs/conda.yaml\
+                                   -P fpath=$PWD/airline_100000.orc
```

Comment:
looks good, but do we provide airline_100000.orc? I included `airline_small.parquet` in the bucket.. maybe that was a bad name though

Reply:
Good catch. Updating now.